content/en/Integrations/webhooks.md

```diff
@@ -0,0 +1,92 @@
+<!-- Provide a link to API docs or API use case. -->
+
+## Before You Start
+
+Before you start creating webhooks, complete the configuration in your app.
+
+1. Generate an **authentication token**. We need this token to authorize HTTP POST requests to your URL and connect to your app.
+1. Make sure that the **URL** where you want to receive notifications is valid and your services work properly.
+
+## Webhook Events
+
+When you set up a webhook, you subscribe to the following events related to your pentests.
+
+| Asset | Pentest | Finding |
+|---|---|---|
+| <li>Asset created</li><li>Asset updated</li><li>Asset deleted</li> | <li>Pentest created</li><li>[Pentest state](/penteststates/) changed</li><li>Pentest deleted</li> | <li>Finding created</li><li>Comment created for a finding</li><li>Finding state changed</li><li>Finding deleted</li>
+
+## Configure Webhooks in the UI
+
+Let’s configure webhooks in the Cobalt app.
+
+### Create a Webhook
+
+To create a webhook:
+
+1. Navigate to the **Integrations** page, and then select **Webhooks** under **Native Integrations**.
+1. On the **Webhooks** page, select **Create Webhook**.
+1. In the overlay that appears, specify the following:
+   - **Webhook Name**
+   - **Webhook URL**: URL to which Cobalt sends HTTP POST requests for pentest [events](#webhook-events).
+     - Use a unique name and URL for each webhook you create.
+   - **Authentication Token**: Your [token](#before-you-start) required to authorize API requests that Cobalt sends to your URL.
+1. When ready, select **Save**.
+1. We send a test API request to the specified URL to validate your webhook. The webhook becomes active once the validation is complete.
+   - If the validation fails, we’ll disable your webhook within 24 hours. See [Troubleshooting](#troubleshooting) for more information.
+
+![Create a webhook in the Cobalt UI](/gsg/CreateWebhook.png "Create a webhook in the Cobalt UI")
+
+### Manage Webhooks
+
+Now you can manage the webhooks that you [created](#create-a-webhook).
+
+- To **enable** or **disable** a webhook, use the toggle under **Active**.
+- To **edit** a webhook, select the three-dot icon under **Actions**, and then select **Edit Webhook**. Update webhook parameters in the overlay that appears.
+- To **delete** a webhook, select the three-dot icon under **Actions**, and then select **Delete Webhook**.
+  - You can disable a webhook without deleting it.
+
+## Troubleshooting
+
+You can check the status of your webhooks on the **Webhooks** page. The following icons indicate that there is a problem with your webhook:
+
+- **Yellow warning**: The webhook stopped responding to events at the specified time. We’ll disable the webhook after 48 hours of failed attempts.
+- **Red error**: The webhook stopped responding, and we disabled it after 48 hours of failed attempts.
+
+Find solutions for common troubleshooting problems in the table below.
```

Comment:
I'm "used to" the [Google Developer Style Guide](https://developers.google.com/style/tables#introductory-sentences-for-tables) practice to intro a table with a colon -- but I realize Microsoft's guide is different in this area.

```suggestion
Find solutions for common troubleshooting problems in the table below:
```